registration.html

```diff
@@ -53,7 +53,7 @@
 
 			<div id="topbar">
 				<div id="topcontainer">
-					<a href="./registration.html"><b class="font-effect-outline">Registration is OPEN!</b></a>
+					<a href="./registration.html"><b class="font-effect-outline">Registration is CLOSED!</b></a>
```

Comment:
I'd change the rest of the pages to ```closed``` also.

Also, possibly change the color of ```#topbar``` in [styles.css](https://github.com/WSU-LUG/wsucon-static/blob/master/scripts/styles.css) to ```darkgray``` or ```orange```.  (comment the old one so it's there for next year.